src/DynamoCoreWpf/Views/PackageManager/Pages/PublishPackageFinishPage.xaml.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using Dynamo.PackageManager;
+using Dynamo.PackageManager.UI;
+
+namespace Views.PackageManager.Pages
+{
+    /// <summary>
+    /// Interaction logic for PublishPackageFinishPage.xaml
+    /// </summary>
+    public partial class PublishPackageFinishPage : Page
+    {
+        private PublishPackageViewModel PublishPackageViewModel;
+
+        public PublishPackageFinishPage()
+        {
+            InitializeComponent();
+
+            this.DataContextChanged += PublishFinishPage_DataContextChanged;
+            this.Tag = "Publish Finished Page";
```

Comment:
Add this to the resources, so that it can be localized.

Reply:
This is just an internal Tag, it's not meant to be shown on UI, I don't think we should localize Tags.